### DISCUSSION TITLE
📣 ODS iOS library v1.0.0 is here!

### DISCUSSION THREAD

**Discussion by pylapp** [Orange-OpenSource/ods-ios#783] [Announcements]
### API changes

- Display accessibility status in menu item
- _Recirculation module_ is independant now and not included anymore in _About module_

### Features

- Add _ODSEmptyState module_
- List of third-party librairies integrated in the app
- Display build tag in about screen for beta releases (TestFlight)

### Fixes

- Accessibility issues
- Various bugs

### Changes

- Update dependencies
- CI/CD migration
- Update assets

## Changelog

https://github.com/Orange-OpenSource/ods-ios/blob/1.0.0/CHANGELOG.md

## Get demo app

https://boosted.orange.com/download-app/

**Comment by pylapp** [comment]
cc @B3nz01d @pya35 @ludovic35 @mccart77

**Reply by pya35** [reply]
Congratulations to the team !
